frontend/packages/kubevirt-plugin/src/components/vms/vmDetailsItemTemplate.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { TemplateModel } from '@console/internal/models';
+import { TemplateKind } from '@console/internal/module/k8s';
+import {
+  useK8sWatchResource,
+  WatchK8sResource,
+} from '@console/internal/components/utils/k8s-watch-hook';
+import {
+  TEMPLATE_TYPE_BASE,
+  TEMPLATE_TYPE_LABEL,
+  TEMPLATE_TYPE_VM,
+  TEMPLATE_VM_COMMON_NAMESPACE,
+} from '../../constants';
+import { getBasicID, prefixedID } from '../../utils';
+import { VMTemplateLink } from '../vm-templates/vm-template-link';
+import VMDetailsItem from './vmDetailsItem';
+
+const VMDetailsItemTemplate: React.FC<VMDetailsItemTemplateProps> = ({ name, namespace }) => {
+  const { t } = useTranslation();
+  const templatesResource: WatchK8sResource = {
+    isList: false,
+    optional: true,
+    kind: TemplateModel.kind,
+    name,
+    namespace,
+    selector: {
+      matchLabels: {
+        [TEMPLATE_TYPE_LABEL]:
+          namespace === TEMPLATE_VM_COMMON_NAMESPACE ? TEMPLATE_TYPE_BASE : TEMPLATE_TYPE_VM,
```

Comment:
since you have name and namespace, there's no need to match by label too. BTW in theory you can have user template in openshift namespace too.